src/util/classes.ts

```diff
@@ -48,27 +86,27 @@ export const defaultCategories: Category[] = [
   {
     name: ['Media', 'Games'],
     rule: { type: 'regex', regex: 'Minecraft|RimWorld' },
-    data: { color: '#F80' },
+    data: { color: COLOR_RED },
   },
   {
     name: ['Media', 'Video'],
     rule: { type: 'regex', regex: 'YouTube|Plex|VLC' },
-    data: { color: '#F33' },
+    data: { color: COLOR_RED },
   },
   {
     name: ['Media', 'Social Media'],
     rule: {
       type: 'regex',
-      regex: 'reddit|Facebook|Twitter|Instagram|devRant',
+      regex: 'reddit|Facebook|Twitter|Instagram|devRant|LinkedIn',
       ignore_case: true,
     },
-    data: { color: '#FCC400' },
+    data: { color: COLOR_RED },
```

Comment:
Was yellow, now red.

Reply:
I can revert it back, but I think the dark red looks ok:

<img width="208" alt="image" src="https://user-images.githubusercontent.com/150855/173603281-037d4f21-a501-4372-bc79-87f7d7b43ff3.png">

Maybe just personal preference on my end, so happy to revert if you'd like.